src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -0,0 +1,157 @@
+      "SELECT c.*, i.institution_name, u.displayname AS researcher, dar.id AS dar_id, dar.reference_id AS dar_reference_id, dar.collection_id AS dar_collection_id, " +
+          "dar.draft AS dar_draft, dar.user_id AS dar_userId, dar.create_date AS dar_create_date, " +
+          "dar.sort_date AS dar_sort_date, dar.submission_date AS dar_submission_date, " +
+          "dar.update_date AS dar_update_date, (dar.data #>> '{}')::jsonb AS data, " +
+          "(dar.data #>> '{}')::jsonb ->> 'projectTitle' as projectTitle " +
+      "FROM dar_collection c " + 
+      "INNER JOIN dacuser u ON u.dacuserid = c.create_user_id " +
+      "LEFT JOIN institution i ON i.institution_id = u.institution_id " +
+      "INNER JOIN data_access_request dar ON c.collection_id = dar.collection_id ";
+  
+  final String filterQuery = 
+    "WHERE COALESCE(i.institution_name, '') ~* :institutionSearchTerm " +
+    "AND (dar.data #>> '{}')::jsonb ->> 'projectTitle' ~* :projectSearchTerm " +
+    "AND u.displayname ~* :researcherSearchTerm " +
+    "AND c.dar_code ~* :darCodeSearchTerm " + 
+    "AND EXISTS " +
+        "(SELECT FROM jsonb_array_elements((dar.data #>> '{}')::jsonb -> 'datasets') dataset " +
+        "WHERE dataset ->> 'label' ~* :datasetSearchTerm)";
+
+  final String projectSortFilterQuery = 
+    getCollectionAndDars + filterQuery + 
+    "ORDER BY <sortField> <sortOrder> " +
+    "LIMIT :limit OFFSET :offset";
+  
+  /**
+   * Find all DARCollections with their DataAccessRequests
+   *
+   * @return List<DarCollection>
+   */
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = DataAccessRequest.class, prefix = "dar")
+  @UseRowReducer(DarCollectionReducer.class)
+  @SqlQuery(
+    " SELECT c.*, dar.id AS dar_id, dar.reference_id AS dar_reference_id, dar.collection_id AS dar_collection_id, " +
+        "dar.draft AS dar_draft, dar.user_id AS dar_userId, dar.create_date AS dar_create_date, " +
+        "dar.sort_date AS dar_sort_date, dar.submission_date AS dar_submission_date, " +
+        "dar.update_date AS dar_update_date, (dar.data #>> '{}')::jsonb AS data " +
+        "FROM dar_collection c, data_access_request dar " +
+    " WHERE c.collection_id = dar.collection_id"
+  )
+  List<DarCollection> findAllDARCollections();
+
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = DataAccessRequest.class, prefix = "dar")
+  @UseRowReducer(DarCollectionReducer.class)
+  @SqlQuery(" SELECT c.*, dar.id AS dar_id, dar.reference_id AS dar_reference_id, dar.collection_id AS dar_collection_id, "
+      + "dar.draft AS dar_draft, dar.user_id AS dar_userId, dar.create_date AS dar_create_date, "
+      + "dar.sort_date AS dar_sort_date, dar.submission_date AS dar_submission_date, "
+      + "dar.update_date AS dar_update_date, (dar.data #>> '{}')::jsonb AS data "
+      + "FROM dar_collection c, data_access_request dar " 
+      + "WHERE c.collection_id = dar.collection_id "
```

Comment:
I think this is better expressed as an inner join like you've done in other queries:
```
    "FROM dar_collection c " +
    "INNER JOIN data_access_request dar ON c.collection_id = dar.collection_id " +
```